code/game/machinery/doors/airlock.dm

```diff
@@ -72,6 +72,7 @@
 	var/hasShocked = 0 //Prevents multiple shocks from happening
 	autoclose = 1
 	var/obj/item/device/doorCharge/charge = null //If applied, causes an explosion upon opening the door
+	var/obj/item/weapon/note //Any papers pinned to the airlock
```

Comment:
>So a photo looks like paper?
Bad.

It seems like it'd be more polite to have explained "hey, it seems weird that photos don't actually look like photos, could you make some for those?" I don't want to get in a flame war with you, but it seems like you were a little abrasive about it, is all.